examples/argocd/baseline/values.yaml

```diff
@@ -0,0 +1,15 @@
+redis-ha:
+  enabled: false
+
+configs:
+  repositories:
+    github-podinfo:
+      name: podinfo
+      url: https://github.com/stefanprodan/podinfo.git
+      username: ""
+      password: ""
+    github-tracerbullet:
+      name: tracerbullet
+      url: https://github.com/radiusmethod/tracerbullet.git
+      username: ""
+      password: ""
```

Comment:
I would rather keep the tests simpler and not have tracerbullet in it (since, while simple, it would be one differentiator from other tests that use podinfo that might introduce flakes (our tests are pretty tight especially on disk, and additional images or charts could cause issues)

Reply:
removed